src/formly/types/templates/multiSelectImage.tpl.html

```diff
@@ -11,14 +11,15 @@
     <div class="content">
         <div class="row">
             <div ng-repeat="col in options.templateOptions.cols" class="{{::options.templateOptions.colClass}}">
-               <div class="multiselect-image {{::element.css}} {{options.data.isActive(element.value) ? 'alert-success' : 'alert-default'}}"
+               <div class="multiselect-image {{::element.css}} {{options.data.isActive(element.value) ? 'alert-success' : 'alert-default'}} form-control"
                      role="alert" ng-repeat="element in col"
                      ng-class="{'disabled': options.data.isDisabled()}"
                      ng-click="options.data.isDisabled() || addRemoveModel(element.value)">
                     <span>{{::element.label}}</span>
                     <i class="fa fa-check-circle" aria-hidden="true"></i>
                 </div>
             </div>
+            <input type="hidden" ng-model="model">
```

Comment:
Some commend would be aproperiate here - to inform why such a monstrous hack was put in place